barretenberg/cpp/src/barretenberg/common/parallel_for_queued.cpp

```diff
@@ -9,101 +10,6 @@
 #include <thread>
 #include <vector>
 
-namespace {
-class ThreadPool {
-  public:
```

Comment:
Why change this file?

Reply:
Ah I see - any reason we went with the thread pool we weren't using? We use the mutex one

Reply:
Yeah, I actually need the queueing function. Batch inserting into the tress will produce a variable number of jobs to be done.